spec/api/v2/schemas/metadata.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    module Schemas
+      module Metadata
+        UUID = { type: :string, format: :uuid }.freeze
+
+        METADATA = {
+          type: :object,
+          properties: {
+            total: {
+              type: :number,
+              examples: [1, 42, 770],
+              description: 'Total number of items'
+
+            },
+            limit: {
+              type: :number,
+              maximum: 100,
+              minimum: 1,
+              default: 10,
+              examples: [10, 100],
+              description: 'Maximum number of items to return'
+            },
+            offset: {
+              type: :number,
+              minimum: 0,
+              default: 0,
+              examples: [15, 90],
+              description: 'Offset of the first item returned'
```

Comment:
https://github.com/RedHatInsights/compliance-backend/pull/1765/files#diff-1e8cf3de001b876faaf2655349c90959c6a725c8ead4744a692aa285e986a4ffR110

inconsistency...please check the rest of the params as well

Reply:
See [latest changes](https://github.com/RedHatInsights/compliance-backend/compare/ee4ccd88988d1e477561e6cee4ce92d9e5cb3780..a533924bdcf94b4de1291330dd855f69d5e93f09)